acceptance/framework/vault/helpers.go

```diff
@@ -190,8 +190,11 @@ func ConfigurePKICA(t *testing.T, vaultClient *vapi.Client) {
 
 // ConfigurePKICertificates configures roles in Vault so that Consul server TLS certificates
 // can be issued by Vault.
-func ConfigurePKICertificates(t *testing.T, vaultClient *vapi.Client, consulReleaseName, ns, datacenter string) string {
+func ConfigurePKICertificates(t *testing.T, vaultClient *vapi.Client, consulReleaseName, ns, datacenter string, maxTtl string) string {
 	// Create the Vault PKI Role.
+	if maxTtl == "" {
+		maxTtl = "1h"
+	}
```

Comment:
I think it might be better to not have a default and just always pass a value in. Given that the alternative would be to have a bunch of tests that pass `""` in. I think that could be confusing.

I think the counter to that argument would be if you want to change this default later, you'd have to change it in several places.

Reply:
I agree.  We don't use it that often that I think it's fine to make it explicit and remove the default.  It's clearer to review.

Reply:
Cool